packages/dapp/src/components/Modal.tsx

```diff
@@ -0,0 +1,19 @@
+
+import { useState, FC } from 'react';
+import { Box, Button, Layer } from 'grommet';
+
+
+export const Modal:FC<{close():void}> = ({close, children}) => {
+  return (
+    <Box>
+      <Layer
+        onEsc={() => close()}
+        onClickOutside={() => close()}
+      >
+        <Box pad='medium'>
+          {children}
+        </Box>
+      </Layer>
+    </Box>
+  );
+}
```

Comment:
It is recommended to configure VS Code to automatically add a `newline` at the end of every file (can be done via config `"files.insertFinalNewline": true`)
Also please install the `Trailing Spaces` plugin to remove all tralling spaces.

Reply:
:heavy_check_mark: